lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -163,11 +163,27 @@
                 display: inline-block;
               }
 
-              .info-date-block {
-                @extend %t-title7;
-
-                color: $gray; // WCAG 2.0 AA compliant
+              .info-date-block-container {
                 display: block;
+
+                .info-date-block{
+                  @extend %t-title7;
+
+                  color: $gray; // WCAG 2.0 AA compliant
```

Comment:
choosing to leave this until we extend the theme-colors to add a color that is as light as possible while passing a11y tests (roughly #767676)